src/main/scss/components/_buttons.scss

```diff
@@ -16,6 +16,7 @@
   margin: 0;
   padding: 0.5rem 0.9rem;
   font-size: 0.875rem;
+  font-weight: normal;
```

Comment:
Annoyingly links and buttons with the class `jenkins-button` have different weights, this fixes that.